docs/manpages/wforce.1.md

```diff
@@ -98,7 +98,7 @@ started with the -c option.
   sinks. For example: 
 
 		> showNamedReportSinks()
-		Name            Address                             Sucesses  Failures
+		Name            Address                             Successes Failures
```

Comment:
I'm not guaranteeing that the alignment is correct, but, the source code did show that the correct spelling was used in https://github.com/PowerDNS/weakforced/blob/40c18eea5692303d2da76910484b942edf07347d/wforce/wforce-lua.cc#L406-L407

As noted earlier, this file is the source for the earlier column changes (it's the block preceding this section).